messaginginapp/src/main/java/io/customer/messaginginapp/ui/InlineInAppMessageBaseView.kt

```diff
@@ -19,7 +20,7 @@ abstract class InlineInAppMessageBaseView @JvmOverloads constructor(
     @AttrRes defStyleAttr: Int = 0,
     @StyleRes defStyleRes: Int = 0
 ) : InAppMessageBaseView(context, attrs, defStyleAttr, defStyleRes) {
-    internal var viewListener: InlineInAppMessageViewListener? = null
+    internal var viewCallback: InlineInAppMessageViewListener? = null
```

Comment:
There are quite a few listeners now and it was getting a bit confusing. So I renamed this to `callback` for clarity. In later PR in stack, the class is also renamed to `Callback` to stay consistent.